app/scripts/modules/core/src/api/ApiService.ts

```diff
@@ -164,9 +164,9 @@ export class API {
   private static init(urls: string[]) {
     const config: IRequestConfig = {
       method: '',
-      url: SETTINGS.gateUrl,
+      url: SETTINGS.gateUrl.toString().replace(/\/+$/, ''),
```

Comment:
omit this one, the one below is great!

Reply:
done (and moved down to the baseUrl getter)